AEPServices/Sources/storage/FileSystemNamedCollection.swift

```diff
@@ -126,6 +126,7 @@ class FileSystemNamedCollection: NamedCollectionProcessing {
             let adobeBaseUrl = baseUrl.appendingPathComponent(adobeDirectory, isDirectory: true)
             do {
                 try fileManager.createDirectory(at: adobeBaseUrl, withIntermediateDirectories: true)
+                try fileManager.setAttributes([FileAttributeKey.protectionKey: FileProtectionType.none], ofItemAtPath: adobeBaseUrl.path)
             } catch {
                 Log.error(label: adobeDirectory, "Failed to create storage directory with error: \(error)")
```

Comment:
Should we consider adjusting this error message? Is it possible to check the reason and avoid printing this message when the error is a result of the directory already existing?

Reply:
I believe it doesn't error when the directory already exists

Reply:
https://github.com/adobe/aepsdk-core-ios/pull/966#discussion_r1318920171